src/content/en/updates/2017/12/chrome-64-deprecations.md

```diff
@@ -0,0 +1,70 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the deprecations and removals in Chrome 64 to help you plan. In this version, performance API changes, removal of support for multiple shadow roots, and removal of a webkit API.
+
+{# wf_updated_on: 2017-12-14 #}
+{# wf_published_on: 2017-12-14 #}
+{# wf_tags: deprecations,removals,chrome64 #}
+{# wf_blink_components: Blink #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: A round up of the deprecations and removals in Chrome 64 to help you plan. In this version, performance API changes, removal of support for multiple shadow roots, and removal of a webkit API. #}
+
+# Deprecations and removals in Chrome 64 {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+In nearly every version of Chrome, we see a significant number of updates and
+improvements to the product, its performance, and also capabilities of the Web
+Platform. This article describes some of the deprecations and removals in Chrome
+64, which is in beta as of December 14. Visit the
+[deprecations page](/web/updates/tags/deprecations) for more deprecations and
+removals from this and previous versions of Chrome. This list is subject to
+change at any time.
+
+## Remove chrome.loadTimes()
+
+The `chrome.loadTimes()` method is a non-standard API that enables collecting
+loading-related metrics in real-world operations. A [Resource Timing Level
+1](https://www.w3.org/TR/resource-timing-1/) API has existed for several years
+while the current version of it is a W3C Candidate Recommendation. Chrome itself
+has had some version of an implementation [since Chrome
```

Comment:
While some of the values from `chrome.loadTimes()` have been available in Chrome for a while, it's only very recently that all of them have been available. I'm not sure what we gain by mentioning this.

Reply:
I'm trying to emphasize our spec compliance.

Reply:
I've reworded though.

Reply:
PTAL